rust/connlib/clients/shared/src/eventloop.rs

```diff
@@ -340,7 +340,9 @@ where
             ErrorReply::UnmatchedTopic => {
                 self.portal.join(topic, ());
             }
-            ErrorReply::NotFound | ErrorReply::Other => {}
+            reason @ (ErrorReply::InvalidVersion | ErrorReply::NotFound | ErrorReply::Other) => {
+                tracing::debug!(%req_id, %reason, "Request failed");
```

Comment:
Don't we want this to be a `warn!`? Or are these cases more common than I'm assuming?

This is the error returned by the portal when the Client connects, not when selecting a Gateway right?

Reply:
This is currently not very well modelled in `phoenix-channel`. We only have one enum that captures all errors for replies when in reality, they can occur at different times. The `InvalidVersion` variant here shouldn't ever happen in production because we catch that at a different place already. Hence these are only `debug!` just so that they don't go completely unnoticed when you are debugging something.

Reply:
👍